### ISSUE TITLE
Run checkAlignment procedure on first render

### ISSUE THREAD

**Issue by yakunins** [signavio/react-stick#113]
Hello, Philipp!

It would be nice to have autoflipping work not only on `handleScroll` but straight ahead after render. So, sticky element will be fully visible even without scrolling. This will help in sutuations when anchor is placed from the start near the edge of the viewport.

For, example this may be implemented by adding such a piece of code before [line 76](https://github.com/signavio/react-stick/blob/master/src/Stick.js#L76) in Stick.js:

```
  useEffect(() => {
    if (!nodeRef.current || !anchorRef.current) {
      return
    }

    checkAlignment(nodeRef.current, anchorRef.current)
  }, [nodeRef.current, anchorRef.current])
```

**Comment by yakunins** [comment]
@frontendphil May I create a PR for this issue?

**Comment by frontendphil** [comment]
Hey, @yakunins sorry for overlooking this for so long! Please go ahead and put together a PR if you already have an idea how to solve this :)

**Comment by yakunins** [comment]
@frontendphil Please take a look ont this PR: https://github.com/signavio/react-stick/pull/159

**Comment by crecotun** [comment]
Closed by #159
